test/e2e/doc.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 The Operator-SDK Authors
```

Comment:
This file isn't necessary, as everything in `test/` is for SDK testing and not exported.

Reply:
well, I had some errors reported by VSCode because there were no non-test go files in those packages, hence these 2 `doc.go` files, but I can remove them if you prefer.

Reply:
done in 2dfd90f